src/services/DiscordService.ts

```diff
@@ -0,0 +1,195 @@
+import profiles from 'decentraland-gatsby/dist/utils/loader/profile'
+import { ChannelType, Client, EmbedBuilder, GatewayIntentBits } from 'discord.js'
+
+import { capitalizeFirstLetter } from '../clients/utils'
+import { profileUrl } from '../entities/Profile/utils'
+import { ProposalType } from '../entities/Proposal/types'
+import { isGovernanceProcessProposal, proposalUrl } from '../entities/Proposal/utils'
+import UpdateModel from '../entities/Updates/model'
+import { UpdateAttributes } from '../entities/Updates/types'
+import { getPublicUpdates, updateUrl } from '../entities/Updates/utils'
+
+const CHANNEL_ID = process.env.DISCORD_CHANNEL_ID
+const TOKEN = process.env.DISCORD_TOKEN
+
+const client = new Client({ intents: [GatewayIntentBits.Guilds] })
+
+const DEFAULT_AVATAR = 'https://decentraland.org/images/male.png'
+const BLANK = '\u200B'
+const PREVIEW_MAX_LENGTH = 140
+
+type Field = {
+  name: string
+  value: string
+}
+
+enum MessageColors {
+  NEW_PROPOSAL = 0x0099ff,
+  FINISH_PROPOSAL = 0x8142f5,
+  NEW_UPDATE = 0x00ff80,
+}
+
+type EmbedMessageProps = {
+  title: string
+  proposalType?: ProposalType
+  description?: string
+  choices: Field[]
```

Comment:
I'd rename `choices` to `fields`. As they are not always choices (like in the update method)